share/fstab

```diff
@@ -1,19 +1,9 @@
 # /etc/fstab: static file system information.
-#
```

Comment:
Looks like you're dropping the empty line here between the `/etc/fstab` title and the next comment. Please restore that.

Reply:
This still looks wrong? Look at the first patch:

```diffdiff --git a/share/fstab b/share/fstab
index 62dd22e5..7fd7852e 100644
--- a/share/fstab
+++ b/share/fstab
@@ -1,10 +1,4 @@
 # /etc/fstab: static file system information.
-#
-# noatime turns off atimes for increased performance (atimes normally aren't 
-# needed); notail increases performance of ReiserFS (at the expense of storage 
-# efficiency).  It's safe to drop the noatime options if you want and to 
-# switch between notail / tail freely.
-#
 # The root filesystem should have a pass number of either 0 or 1.
 # All other filesystems should have a pass number of 0 or greater than 1.
 #
```

You're removing two empty ("#") lines, one before and one after the comment block you're removing. You should only remove one so that there's still separation between the title line ("/etc/fstab: static file") and the next comment block ("The root filesystem ...").

Reply:
Fixed on merge.